metadata.yaml

```diff
@@ -17,14 +17,11 @@ resources:
     filename: promethues-bind.exporter.snap
 
 requires:
-  juju-info:
-    interface: juju-info
-    scope: container
-
-provides:
   designate-bind:
     interface: bind-exporter
```

Comment:
I think this interface should be named "bind-client", because in the future, a hostname and port could be passed via the relation (not for now, since the info is hardcoded, `hostname=127.0.0.1, port=8053`). The endpoint name could be named "bind-stats" instead of `designate-bind`. This charm could potentially work with any charm that supports the bind service (e.g. a charm unrelated to OpenStack and Designate).

Reply:
I was thinking of some generic name for the endpoint, but I couldn't find any other charm that could use it.
:+1: